lit_llama/model.py

```diff
@@ -13,7 +13,7 @@
 
 @dataclass
 class LLaMAConfig:
-    block_size: int = 4096
+    block_size: int = 2048
```

Comment:
as in https://github.com/facebookresearch/llama/blob/57b0eb62de0636e75af471e49e2f1862d908d9d8/llama/model.py#L30